samples/sensor/bmi160/dts_fixup.h

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2018 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef CONFIG_BMI160_NAME
+#define CONFIG_BMI160_NAME		""
```

Comment:
I think this workaround is dangerous for samples because users may incorrectly think that the sample should work on their board. The compile will succeed, but the application will fail at runtime. We really should only be building this sample for boards that support the sensor.

Reply:
Ok, changed to only run this on the arduino_101_sss